modules/internal/BytesStringCommon.chpl

```diff
@@ -183,6 +184,65 @@ module BytesStringCommon {
     return ret;
   }
 
+   /*
```

Comment:
This funcition will not be documented, but it is still a good idea to pay some attention to the comment here. The part about the arguments should be reformatted to say something like:

```chapel
/* ...

   :arg buf: ....

   :arg buflen: ...
   etc. etc
*/
See documentation in String.chpl or Bytes.chpl for examples